### PR TITLE
add new constructors that receives actor addr

### PR DESCRIPTION
**Pull request by uchida**
adds New API: from actor for session backend as same as https://github.com/Idein/actix-redis-cluster/pull/12

### CODE REVIEW

File: actix-redis/src/session.rs [hunk 1]
```diff
@@ -30,13 +30,21 @@ pub struct RedisSession<R: Actor = RedisActor>(Rc<Inner<R>>);
...
+    /// Create new redis session backend
+    ///
+    /// * `addr` - Addr of the redis actor
+    pub fn from_redis(addr: Addr<RedisActor>, key: &[u8]) -> Self {
         RedisSession(Rc::new(Inner {
             key: Key::from_master(key),
             cache_keygen: Box::new(|key: &str| format!("session:{}", &key)),
             ttl: 7200,
-            addr: RedisActor::start(addr),
+            addr: addr,
```
Comment (pandaman64) [inline actix-redis/src/session.rs]:
```suggestion
            addr,
```

File: actix-redis/src/session.rs [hunk 2]
```diff
@@ -49,15 +57,23 @@ impl RedisSession<RedisActor> {
...
-    /// * `addr` - address of the redis server
-    pub fn new_cluster<S: Into<String>>(addr: S, key: &[u8]) -> Self {
+    /// * `addr` - Addr of the redis cluster actor
+    pub fn from_cluster(addr: Addr<RedisClusterActor>, key: &[u8]) -> Self {
         RedisSession(Rc::new(Inner {
             key: Key::from_master(key),
             cache_keygen: Box::new(|key: &str| format!("session:{}", &key)),
             ttl: 7200,
-            addr: RedisClusterActor::start(addr),
+            addr: addr,
```
Comment (pandaman64) [inline actix-redis/src/session.rs]:
```suggestion
            addr,
```

**Review by pandaman64 — CHANGES_REQUESTED** [review]
Minor nits regarding the style. Otherwise :+1:

File: actix-redis/src/session.rs (hunk 1 above)
Comment (uchida) [reply actix-redis/src/session.rs]: :pray: done

File: actix-redis/src/session.rs (hunk 2 above)
Comment (uchida) [reply actix-redis/src/session.rs]: :pray: done

**Pull request merged**